internal/provider/tags_data_source.go

```diff
@@ -87,6 +88,14 @@ func (d *TagsDataSource) Read(ctx context.Context, req datasource.ReadRequest, r
 		return
 	}
 
+	if _, set := os.LookupEnv("TF_APKO_DISABLE_VERSION_TAGS"); set {
+		resp.Diagnostics.AddWarning("Version tags disabled", "Version tags are disabled using TF_APKO_DISABLE_VERSION_TAGS environment variable")
+		data.Tags = []string{}
+		data.Id = types.StringValue("")
```

Comment:
I seem to recall `id: ""` meaning something _special_ in TF, but I don't remember what that is. Should I make up something else here? I can also set the ID to what the tags _would_ have been, and just not set the output.

Reply:
Went with `id=disabled`, added tests.